src/server/pps/server/master.go

```diff
@@ -139,7 +139,7 @@ func newMaster(ctx context.Context, env Env, etcdPrefix string, kd InfraDriver,
 // pipelines are created/removed.
 func (a *apiServer) master(ctx context.Context) {
 	masterLock := dlock.NewDLock(a.env.EtcdClient, path.Join(a.etcdPrefix, masterLockPath))
-	backoff.RetryUntilCancel(ctx, func() error { //nolint:errcheck
+	backoff.RetryUntilCancel(ctx, func() error {
```

Comment:
I wonder if a function which retries infinitely and has no return value would be useful so we don’t have a ton of `nolint:errcheck` throughout. It’s our own backoff package, so it would be easy to add it there and then rewrite all usage, right?